docs/explanation/state-transition-testing.md

```diff
@@ -192,13 +192,12 @@ emit multiple events in a single charm execution.
 
 ## The virtual charm root
 
-Before executing the charm, the framework copies the charm's `/src`, any libs,
-and the metadata, config, and actions YAML to a temporary directory. The charm
-will see that temporary directory as its 'root'. This allows us to keep things
-simple when dealing with metadata that can be either inferred from the charm
-type being passed to `Context` or be passed to it as an argument, thereby
-overriding the inferred one. This also allows you to test charms defined on the
-fly, as in:
+Before executing the charm, the framework writes the metadata, config, and actions
+YAML to a temporary directory. The charm will see that temporary directory as its
```

Comment:
my 2c: either "YAML files" or list the file names with the extension.

Reply:
```suggestion
YAML files to a temporary directory. The charm will see that temporary directory as its
```